src/components/videos/VideosList.test.js

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react';
+import VideosList from './VideosList';
+import mockData from '../../helper/mock-data.json';
+
+describe('VideosList Component', () => {
+  test('should not render nothing if props.list is empty', () => {
+    render(<VideosList list={[]} />);
+    expect(screen.queryByRole('listitem')).toBeFalsy();
+  });
+
+  test('should render at least one item if props.list is not empty', () => {
+    render(<VideosList list={[mockData.items[1]]} />);
+    expect(screen.queryByRole('listitem')).toBeTruthy();
```

Comment:
It is not recommended to use `queryBy` selectors when not testing for no existence.  [source](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence)

Same comment as above, this would be more meaningful is using `.toBeInTheDocument` instead of `toBeTruthy`

Reply:
Okay thank you very much!!